Zend/zend_attributes_arginfo.h

```diff
@@ -134,12 +134,12 @@ static zend_class_entry *register_class_Attribute(void)
 	zend_declare_typed_property(class_entry, property_flags_name, &property_flags_default_value, ZEND_ACC_PUBLIC, NULL, (zend_type) ZEND_TYPE_INIT_MASK(MAY_BE_LONG));
 	zend_string_release(property_flags_name);
 
-	zend_string *attribute_name_Attribute_class_Attribute = zend_string_init_interned("Attribute", sizeof("Attribute") - 1, 1);
-	zend_attribute *attribute_Attribute_class_Attribute = zend_add_class_attribute(class_entry, attribute_name_Attribute_class_Attribute, 1);
-	zend_string_release(attribute_name_Attribute_class_Attribute);
-	zval attribute_Attribute_class_Attribute_arg0;
-	ZVAL_LONG(&attribute_Attribute_class_Attribute_arg0, ZEND_ATTRIBUTE_TARGET_CLASS);
-	ZVAL_COPY_VALUE(&attribute_Attribute_class_Attribute->args[0].value, &attribute_Attribute_class_Attribute_arg0);
+	zend_string *attribute_name_Attribute_class_Attribute_1 = zend_string_init_interned("Attribute", sizeof("Attribute") - 1, 1);
```

Comment:
Duplicate attributes are rate. I propose to not add `_1` if there are no duplicates.

If they are, 0-based indexing looks more natural to me, as everything in PHP is 0-based indexed.

Reply:
Conditionally adding/not adding the attributes would complicate the very complicated code even more. Since these files are not intended to be human readable (the git attribute change was pushes inadvertently, which I'll remove right now), I don't see much reason about using either an index-less, or 0-based indices. But other reviewers can tell their preference.

Reply:
At last, I went with 0-based indices as I don't really care which style is used.